### DISCUSSION TITLE
Seria possível desativar o gale?

### DISCUSSION THREAD

**Discussion by LuizHenryBR** [elizandrodantas/bot-blaze-telegram#27] [Q&A]
O bot tem uma taxa de acerto muito alta de forma direta (sem gale) eu gostaria de desativar o gale e testar mas não sei como.

**Comment by elizandrodantas** [comment]
https://github.com/elizandrodantas/bot-blaze-telegram#todas-op%C3%B5es-com-forma-de-uso todas as opções estão no readme
